.github/workflows/standard_spa_pr.yml

```diff
@@ -375,7 +375,7 @@ jobs:
       && !cancelled()
       && inputs.use_e2e
     needs: [create-shared-vars, magic_url, migration_number, prepare_for_e2e_test]
-    continue-on-error: ${{ inputs.e2e_pass_on_error }}
+    continue-on-error: ${{ inputs.e2e_pass_on_error && !inputs.externally_triggered }}
```

Comment:
Addresses https://jupiterone.atlassian.net/browse/APP-13191, ensuring the proper status is reported to the trigger repo when the job is triggered externally.